src/components/Books.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+
+const Books = ({ books }) => (
+  <>
+    {books.map((book) => (
+      <div className="book" key={book.id}>
```

Comment:
- [ ] It would be nice if we fix the browser console error, here on `line6` we have `key={book.id}` this is where the error comes from, since in our home page `line7-8` we only have a key attribute and not id attribute. So is either, we change to `key={book.key}` or we leave it the way it is and change the code on home page `line7-8` to `title: 'Half of the yellow sun', author: 'Chimamanda Adichie', id: 1`
![Screenshot from 2023-04-03 11-00-53](https://user-images.githubusercontent.com/46388113/229491707-08b49f80-e0d7-41ac-9a14-398f6c18e9bc.png)